src/index.js

```diff
@@ -403,11 +403,13 @@ AFRAME.registerComponent('editor', {
 
 	AFRAME.registerComponent('n-sound', {
 		init: nativeComponentInit,
-		pause: function () {
+		pauseSound: function () {
```

Comment:
play and pause have special meaning in aframe. All components have a base implementation. I tried using them, and a-frame should have wrapped my methods with a call to the base method, but it didn't work, so I just renamed them.